src/app.js

```diff
@@ -520,15 +520,7 @@ var run = function () {
     } else {
       url = 'https://ethereum.github.io/solc-bin/bin/' + version;
     }
-    var isFirefox = typeof InstallTrigger !== 'undefined';
```

Comment:
I was only referring to this issue `Worker error: undefined`.
@chriseth the build is ok if we don't use the worker for firefox. 
the Ballot is included but not yet tested (see the comment in the code).
We could merge this now and fix the worker in another pr.